### ISSUE TITLE
Dynamic Help Should have PSReadlineOption to open in either browser or "-ShowWindow" in addition to pager

### ISSUE THREAD

**Issue by JustinGrote** [PowerShell/PSReadLine#2270]
## Description of the new feature/enhancement
The new dynamic help feature should have a psreadline option to display in the ShowWindow Dialog when pressing the trigger key (F1)
![image](https://user-images.githubusercontent.com/15258962/110184866-bf831400-7dc5-11eb-8526-30cc3679d770.png)

Obviously this would be Windows-Only until a cross-platform GUI is chosen.

It should also have an option to open it in a browser with `-Online` when F1 is pressed, if a online URL is supplied. Same action as `Get-Help Get-Command -Online`

**Comment by theJasonHelmick** [comment]
I agree its interesting, but as you point out, it would need cross platform support. What benefits do you think users have with -ShowWindow style over the existing pager?

**Comment by JustinGrote** [comment]
@theJasonHelmick I probably should rename the issue, `-Online` support would be more important. Basically have a PSReadline option that, if an online URL is available for a help command, start it in the default browser if it is available (which should work xplat). If either of those two conditions don't match, use the default pager.

`-ShowWindow` would work same way, if it's on a platform that doesn't support it, it will just open the normal pager.

The main benefits are search and scrolling vs. the pager.

**Comment by ninmonkey** [comment]
> The main benefits are search and scrolling vs. the pager.

This new keybind ignores my `$Env:Pager` . Is that on purpose for a technical reason?

**Comment by KyleKolander** [comment]
I left the following comment at [#2270](https://github.com/PowerShell/PSReadLine/issues/2271#issuecomment-1030907271) related to this same issue.

@theJasonHelmick - Having to scroll a single line at a time is just painful.  I was really excited about the F1 feature...until I started using it and realized it does not honor the PAGER environment variable and is tightly coupled to a severely limited pager implementation.  Awesome idea, but too inefficient unless I only care to see help about a specific parameter.
